### PR TITLE
Update retention-policies-teams.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Chat and call data for External users

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4083

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/retention-policies-teams.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ For the two paths in the diagram:
 
 When the retention policy is retain-only, or delete-only, the content's paths are variations of retain and delete.
 
+## Teams chat or calls for guest users and external users
+
+All Teams chats and calls are stored as compliance records in the Guest users shadow mailbox. However, Federated users in a call or chat would retain the compliance records in their own tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/retention-policies-teams.md]:
```suggestion
All Teams chats and calls are stored as compliance records in the guest user's shadow mailbox. However, federated users in a call or chat would retain the compliance records in their own tenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by cabailey** [comment]
As per the comments in the related GitHub Issue, closing this PR.

**Pull request closed without merging**